code_of_conduct.md

```diff
@@ -33,42 +33,43 @@ contain:
 
 - sexual or violent imagery
 - exclusionary language
-- language which is not appropriate for an all-ages audience
+- language which is inappropriate for an all-ages audience
 
-For the avoidance of doubt, public presentations, include but are not limited
-to keynotes, presentations, lightning talks, addresses, IRC, mailing list posts
+For the avoidance of doubt, public presentations include but are not limited
+to: keynotes, presentations, lightning talks, addresses, IRC, mailing list posts,
 and forums. If you are in doubt, consider it covered.
 
 If presenters are unsure whether their material is suitable, they are
-encouraged to show it to the meetup host or a meetup organiser before their
+encouraged to show it to the meetup host or an organiser before their
 session.
 
-### Be aware
+### Be Aware
 
 We ask everyone to be aware that we will not tolerate intimidation, harassment,
-or any abusive, discriminatory or derogatory behaviour by anyone at any event
+or any abusive, discriminatory, or derogatory behaviour by anyone at any event,
 or online. Complaints can be made to the [meetup
 organisers](http://www.meetup.com/The-London-Django-Meetup-Group/members/?op=leaders)
-either face to face at a meetup, or via the Meetup website. All complaints
-made to event organisers will remain confidential and be taken seriously. The
-complaint will be treated appropriately with discretion. Should event
-organiser consider it appropriate, measures they may take include:
+either face to face at a meetup, or via the Meetup website. Any complaints
+made to event organisers will remain confidential, be taken seriously, and
+be treated appropriately with discretion. Should an event organiser consider
+it appropriate, the complainee may be:
 
-- being told to apologise
-- being told to stop/modify their behaviour appropriately
-- being warned that enforcement action may be taken if the behaviour continues
-- being asked to leave the venue immediately
-- be prohibited from continuing to attend the event
-- have the incident reported to the appropriate authorities
+- told to apologise
+- told to stop/modify their behaviour appropriately
+- warned that enforcement action may be taken if the behaviour continues
+- asked to leave the venue immediately
+- prohibited from continuing to attend the event
 
-### What does that mean for me?
+When necessary, an incident will be reported to the appropriate authorities.
```

Comment:
I separated this as it made the bullet point list simply a list of actions against complainees, and indicated that is a "when necessary"/last resort kind of action.